tools/tasks/seed/build.html_css.ts

```diff
@@ -64,7 +64,12 @@ function processComponentScss() {
     .pipe(plugins.sass(Config.getPluginConfig('gulp-sass')).on('error', plugins.sass.logError))
     .pipe(plugins.postcss(processors))
     .on('error', reportPostCssError)
-    .pipe(plugins.sourcemaps.write(isProd ? '.' : ''))
+    .pipe(plugins.sourcemaps.write(isProd ? '.' : '', {
+      sourceMappingURL: (file: any) => {
+        // write absolute urls to the map files
+        return `/${file.relative}.map`;
```

Comment:
Maybe it'll break if [`APP_BASE` is changed](https://github.com/mgechev/angular-seed/blob/master/tools/config/seed.config.ts#L113). You can use `Config.APP_BASE` instead of `/`.